client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -49,11 +49,25 @@ export const WpcomPluginInstallButton = props => {
 		}
 	}
 
+	function isDisabled() {
+		// Pressable prevents installation of some plugins, so we need to disable AT for them.
+		// More info here: https://kb.pressable.com/faq/does-pressable-restrict-any-plugins/
+		const unsupportedPlugins = [
+			'nginx-helper',
+			'w3-total-cache',
+			'wp-rocket',
+			'wp-super-cache',
+			'bwp-minify',
+		];
```

Comment:
I think we don't need two copies of this list. The `plugin-install-button` has a `disabled` prop, so we could just do the logic for that in `plugin-meta`?